### ISSUE TITLE
distributed runtime leaking memory on windows

### ISSUE THREAD

**Issue by guschmue** [tensorflow/tensorflow#9867]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: b'unknown' 1.1.0
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**: follow https://www.tensorflow.org/deploy/distributed

### Describe the problem
The memory footprint for parameter server keep growing.
The cause is a memory leak in a windows specific path in grpc which is fixed here:
https://github.com/grpc/grpc/commit/fa242cba900ece728d2910e7396d02ebab4ddb2c
I filed the issue issue so others don't need to spend the time debugging it and as reason to update the grpc version.

**Comment by girving** [comment]
@mrry Looks like we should update grpc if that hasn't already happened.

**Comment by mrry** [comment]
Thanks for figuring out the problem, @guschmue!

I think updating gRPC is blocked by #7466, which appears to be on hold because of a different(?) bug in gRPC on Windows: https://github.com/grpc/grpc/pull/9826.

Handing this off to @jhseu, who is shepherding in #7466.

**Comment by guschmue** [comment]
https://github.com/tensorflow/tensorflow/pull/11768 - closing.
